### DISCUSSION TITLE
.hdr 格式 怎么转成 .bin 的格式

### DISCUSSION THREAD

**Discussion by chengxw** [galacean/discussion#76] [Engine] [answered]
官网demo上没有关于.bin格式转换的说明，只有.env的， 但是.env的加载内存会很大

**Comment by zhuxudong** [comment, marked as answer]
demo 里面使用的 .bin 文件是通过[编辑器](https://galacean.antgroup.com/editor)或者 [glTF Viewer](https://galacean.antgroup.com/#/gltf-viewer) 进行离线烘焙得到的 IBL 烘焙产物

[glTF Viewer](https://galacean.antgroup.com/#/gltf-viewer) 默认烘焙分辨率已改为 128 * 128，大约500KB

**Comment by eyworldwide** [comment]
`.env` 和 `.bin` 是一种东西，只是为了上传到特定的 CDN，所以把 `.env` 后缀改成了 `.bin`
